### DISCUSSION TITLE
Cannot import from my Organization

### DISCUSSION THREAD

**Discussion by darrensw777** [vercel/community#7414] [Account Issue] [answered]
### Question

**Question**
My plan is Hobby, I tried to import a repo which is under an organization that I own in GitHub but get the following error:

"You are attempting to import a Git repository from a GitHub organization, GitLab group, or Bitbucket team. Start a Pro Trial or upgrade to Pro."

When you look at pricing details on the Vercel site it clearly says, for a Hobby account, **Import any git repo**.

Any ideas on how we fix this?

**Comment by pawlean** [comment]
Hi @darrensw777!

I couldn't reproduce this. It does sound like a bug! Can you give me your vercel ID so I can look into this more?

Have you also tried importing different repos to see if it's just a specific repo that gives you this error or just all repo imports?

**Reply by darrensw777** [reply]
Hi! Thanks for the reply, where would I find my Vercel ID? Do you know?

**Reply by pawlean** [reply]
Go to https://vercel.com/account and scroll down to `Vercel ID`. 

> _Have you also tried importing different repos to see if it's just a specific repo that gives you this error or just all repo imports?_

Re-sharing this too 🙂

**Reply by darrensw777** [reply]
Many thanks - it is: 4DJzezX165WR9ES6f3FRbpJv

Yes is the answer to your question, I have imported many before. Never that belonged to an organization though that I know of.

Rgds,

Darren

**Reply by pawlean** [reply]
@darrensw777 So to clarify, this only happens when you import repos from YOUR organisation?

**Reply by darrensw777** [reply]
Hi, yes, that seems correct. I haven't tried to add any from any other organisations though.

**Reply by pawlean** [reply]
@darrensw777 I'm just double-checking this internally, will give you an update shortly.

**Reply by darrensw777** [reply]
@pawlean Many thanks!!

**Reply by pawlean** [reply, marked as answer]
Hi @darrensw777! Thanks for your patience here. I want to confirm that we only support git repositories that are hosted in personal account scope for Hobby users, so you cannot deploy a repository that is created under an organization account.

Apologises for the confusion, we've shared this feedback internally and will make changes to the pricing page to make it clearer. Let me know if you have any questions or if I can help with anything else!

**Reply by darrensw777** [reply]
Hi @pawlean, thanks for the answer, this is very misleading to users though. This was obviously an intended end result yet no-one documented it?

**Reply by pawlean** [reply]
I hear you, @darrensw777! We'll get this changed as soon as we can. Thank you again for the feedback.

**Reply by foinf** [reply]
> Hi @darrensw777! Thanks for your patience here. I want to confirm that we only support git repositories that are hosted in personal account scope for Hobby users, so you cannot deploy a repository that is created under an organization account.
> 
> Apologises for the confusion, we've shared this feedback internally and will make changes to the pricing page to make it clearer. Let me know if you have any questions or if I can help with anything else!

Hi Pawlean. Has this been a recent change? Because I certainly added a organization project to my Hobby Vercel account and it worked (still does) but I am unable to add any new ones. So I am confused about the change and its timeline.

**Reply by sarojkunwar999** [reply]
> > Hi @darrensw777! Thanks for your patience here. I want to confirm that we only support git repositories that are hosted in personal account scope for Hobby users, so you cannot deploy a repository that is created under an organization account.
> > Apologises for the confusion, we've shared this feedback internally and will make changes to the pricing page to make it clearer. Let me know if you have any questions or if I can help with anything else!
> 
> Hi Pawlean. Has this been a recent change? Because I certainly added a organization project to my Hobby Vercel account and it worked (still does) but I am unable to add any new ones. So I am confused about the change and its timeline.

I am also facing the same issue,
Has this been a recent change or i am missing any kind of permission issues

**Reply by pawlean** [reply]
@foinf @sarojkunwar999 This isn't a recent change, no.

**Reply by nilsonaf** [reply]
> @foinf @sarojkunwar999 This isn't a recent change, no.

Hello! And when was it changed? I ask because I was able to import normally 14 days ago. I currently have two imports and I want to import a third one and this same message is appearing "You are attempting to import a Git repository from a GitHub organization. To collaborate with your team members, a Pro subscription is required.". Was it a BUG and has it been fixed? If so, is there any place pointing this out?

**Reply by justinbalaguer** [reply]
facing a same issue, i also was able to import from before, but now it seems like i cant anymore :v

**Reply by samos123** [reply]
I was surprised by this since I created a separate org just for namesake even though it's my personal private repo. I guess I should just move it back to my personal github account for now :(

**Reply by theaafofficial** [reply]
you can fork the repo and deploy it, that's the only workaround I found. Just sync it.

[a comment by an automated account is not shown]
